### ISSUE TITLE
Add support for manifest refs

### ISSUE THREAD

**Issue by marcosnils** [estesp/manifest-tool#265]
This works: `manifest-tool inspect nginx:latest` 

This fails: 
```
 manifest-tool inspect nginx@sha256:88a0a069d5e9865fcaaf8c1e53ba6bf3d8d987b0fdc5e0135fec8ce8567d673e
FATA[0000] image reference must include a tag; manifest-tool does not default to 'latest
```

**Comment by estesp** [comment]
Sorry for the delay! Will be enabled by #266

**Comment by estesp** [comment]
Hey @marcosnils just FYI, I released v2.1.8 that includes this capability

**Comment by marcosnils** [comment]
thx Phil :pray:
